FILE: tryCatch.kt
    public final fun test_1(): R|kotlin/Unit| {
        try {
            lkonst x: R|kotlin/Int| = Int(1)
        }
        catch (e: R|kotlin/RuntimeException|) {
            lkonst y: R|kotlin/Int| = Int(2)
        }
        catch (e: R|kotlin/Exception|) {
            lkonst z: R|kotlin/Int| = Int(3)
        }

    }
    public final fun test_2(): R|kotlin/Unit| {
        lkonst x: R|kotlin/Int| = try {
            Int(1)
        }
        catch (e: R|kotlin/Exception|) {
            Int(2)
        }

    }
    public final fun test_3(b: R|kotlin/Boolean|): R|kotlin/Unit| {
        while(Boolean(true)) {
            try {
                when () {
                    R|<local>/b| ->  {
                        ^test_3 Unit
                    }
                }

                lkonst x: R|kotlin/Int| = Int(1)
                when () {
                    R|<local>/b|.R|kotlin/Boolean.not|() ->  {
                        break@@@[Boolean(true)]
                    }
                }

            }
            catch (e: R|kotlin/Exception|) {
                continue@@@[Boolean(true)]
            }
            catch (e: R|kotlin/RuntimeException|) {
                break@@@[Boolean(true)]
            }

            lkonst y: R|kotlin/Int| = Int(2)
        }

        lkonst z: R|kotlin/Int| = Int(3)
    }
